app/services/schools/advice_page_benchmarks/baseload_benchmark_generator.rb

```diff
@@ -0,0 +1,17 @@
+module Schools
+  module AdvicePageBenchmarks
+    class BaseloadBenchmarkGenerator < SchoolBenchmarkGenerator
+      def benchmark_school
+        return nil unless baseload_service.enough_data?
+        comparison = baseload_service.benchmark_baseload
+        return comparison.category
```

Comment:
Perhaps this can be simplified?
```
      def benchmark_school
        return unless baseload_service.enough_data?

        baseload_service.benchmark_baseload.category
      end
```